test/powershell/Modules/Microsoft.PowerShell.Utility/typedata.tests.ps1

```diff
@@ -1,19 +1,23 @@
 ﻿Describe "TestData cmdlets" -Tags "CI" {
     Context "Get-TypeData" {
         It "System.Array" {
-            (Get-TypeData System.Array).TypeName | Should Be System.Array
+            Get-TypeData System.Array | Should Be $null
```

Comment:
I just add it by analogy to [other](https://github.com/PowerShell/PowerShell/pull/3231/files/fd09cd7804fb760d08d9d39e172554c82f38f1a9#diff-7cd0edfab7610e4c4c8e59d2b167c885L24) existing test. If you think, that it is not necessary here, then I can remove that last commit.